src/Microsoft.DotNet.Wpf/src/PresentationCore/System/Windows/Media/Imaging/BitmapDecoder.cs

```diff
@@ -1039,20 +1039,6 @@ private static void DemandIfImageBlocked()
                 isImageDisabled = new SecurityCriticalDataForSet<bool>(SafeSecurityHelper.IsFeatureDisabled(SafeSecurityHelper.KeyToRead.MediaImageDisable));
```

Comment:
This can be removed as well.  The only uses of these are in this function it seems.  Might as well clean it all out, function and fields included.